### PR TITLE
New Top Bar Icons

### PR DESCRIPTION
**Pull request by PrestonB1123**
Replaced icons in the top bar with new ones and changed the levels meter to be fill up the icon itself instead of a bar on the side.
https://highfidelity.atlassian.net/browse/DEV-136

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19995/HighFidelity-Beta-PR15774-bd78dd9.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19995/HighFidelity-Beta-Interface-PR15774-bd78dd9.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19995/HighFidelity-Beta-PR15774-bd78dd9.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19995/HighFidelity-Beta-Interface-PR15774-bd78dd9.exe

**Comment by PrestonB1123** [comment]
build this please

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19999/HighFidelity-Beta-PR15774-bd78dd9.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19999/HighFidelity-Beta-Interface-PR15774-bd78dd9.dmg

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19999/HighFidelity-Beta-PR15774-bd78dd9.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19999/HighFidelity-Quest-Beta-PR15774-bd78dd9.apk)

**Comment by RyanDowne** [comment]
I don't think the PTT icons look good here. In particular the "off" version. The line through the hand icon makes it hard to parse. They both seem too squat to me 

The mic icon doesn't turn red when it's overloaded and the "!" appears and it should I think. Right now the icon turns to the "!" version in green then goes red for a second and then goes back to normal. I think this should also be reviewed by Joshua. If there is a Jira ticket for this job, I can add these comments as well.

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20004/HighFidelity-Beta-PR15774-538b15d.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20004/HighFidelity-Quest-Beta-PR15774-538b15d.apk)

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20004/HighFidelity-Beta-PR15774-538b15d.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20004/HighFidelity-Beta-Interface-PR15774-538b15d.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20004/HighFidelity-Beta-PR15774-538b15d.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20004/HighFidelity-Beta-Interface-PR15774-538b15d.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20132/HighFidelity-Beta-PR15774-1481c74.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20132/HighFidelity-Beta-Interface-PR15774-1481c74.dmg

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20132/HighFidelity-Beta-PR15774-1481c74.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20132/HighFidelity-Quest-Beta-PR15774-1481c74.apk)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20132/HighFidelity-Beta-PR15774-1481c74.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20132/HighFidelity-Beta-Interface-PR15774-1481c74.exe

File: interface/resources/qml/hifi/simplifiedUI/topBar/SimplifiedTopBar.qml [hunk 1]
```diff
@@ -47,12 +47,6 @@ Rectangle {
 
         onSkeletonModelURLChanged: {
             root.updatePreviewUrl();
-
```
Comment (zfox23) [inline interface/resources/qml/hifi/simplifiedUI/topBar/SimplifiedTopBar.qml]: Why did you remove this code?

File: interface/resources/qml/hifi/simplifiedUI/topBar/SimplifiedTopBar.qml [hunk 2]
```diff
@@ -101,8 +95,7 @@ Rectangle {
                 inventoryFullyReceived = true;
 
                 // If we have an avatar in our inventory AND we haven't already auto-selected an avatar...
-                if ((!Settings.getValue("simplifiedUI/alreadyAutoSelectedAvatar", false) ||
-                    MyAvatar.skeletonModelURL.indexOf("defaultAvatar") > -1 || MyAvatar.skeletonModelURL.indexOf("fst") === -1) && topBarInventoryModel.count > 0) {
+                if (!Settings.getValue("simplifiedUI/alreadyAutoSelectedAvatar", false) && topBarInventoryModel.count > 0) {
```
Comment (zfox23) [inline interface/resources/qml/hifi/simplifiedUI/topBar/SimplifiedTopBar.qml]: Why did you change this code?

File: interface/resources/qml/hifi/simplifiedUI/topBar/SimplifiedTopBar.qml [hunk 3]
```diff
@@ -141,7 +134,7 @@ Rectangle {
         id: avatarButtonContainer
         anchors.verticalCenter: parent.verticalCenter
         anchors.left: parent.left
-        anchors.leftMargin: 16
+        anchors.leftMargin: 2
```
Comment (zfox23) [inline interface/resources/qml/hifi/simplifiedUI/topBar/SimplifiedTopBar.qml]: was this on purpose? these margins seem quite tight to me now

File: interface/resources/qml/hifi/simplifiedUI/topBar/SimplifiedTopBar.qml [hunk 4]
```diff
@@ -210,7 +203,7 @@ Rectangle {
         id: inputDeviceButton
         anchors.verticalCenter: parent.verticalCenter
         anchors.left: avatarButtonContainer.right
-        anchors.leftMargin: 6
+        anchors.leftMargin: 2
```
Comment (zfox23) [inline interface/resources/qml/hifi/simplifiedUI/topBar/SimplifiedTopBar.qml]: was this on purpose? these margins seem quite tight to me now

File: interface/resources/qml/hifi/simplifiedUI/topBar/SimplifiedTopBar.qml [hunk 5]
```diff
@@ -232,9 +225,8 @@ Rectangle {
                 AudioScriptingInterface.systemInjectorGain === simplifiedUI.numericConstants.mutedValue
             source: outputDeviceButton.outputMuted ? "./images/outputDeviceMuted.svg" : "./images/outputDeviceLoud.svg"
             anchors.centerIn: parent
-            width: 20
-            height: 20
-            fillMode: Image.PreserveAspectFit
+            width: outputDeviceButton.outputMuted ? 25 : 26
```
Comment (zfox23) [inline interface/resources/qml/hifi/simplifiedUI/topBar/SimplifiedTopBar.qml]: did you experiment with using `fillMode` instead of having to manually specify `width`? i'd prefer to not see ternary operators on lines like this

File: interface/resources/qml/hifi/simplifiedUI/inputDeviceButton/InputDeviceButton.qml [hunk 6]
```diff
@@ -116,83 +105,86 @@ Rectangle {
     Item {
         id: icon
         anchors.verticalCenter: parent.verticalCenter
-        anchors.right: parent.horizontalCenter
+        anchors.horizontalCenter: parent.horizontalCenter
         anchors.rightMargin: 2
-        width: 13
-        height: 21
+        width: pushToTalk ? 16 : (muted ? 20 : 16)
```
Comment (zfox23) [inline interface/resources/qml/hifi/simplifiedUI/inputDeviceButton/InputDeviceButton.qml]: did you experiment with using `fillMode` instead of having to manually specify `width`? i'd prefer to not see ternary operators on lines like this

File: interface/resources/qml/hifi/simplifiedUI/inputDeviceButton/InputDeviceButton.qml [hunk 7]
```diff
@@ -116,83 +105,86 @@ Rectangle {
...
-            color: micBar.muted ? colors.mutedColor : "white"
+        Item {
             width: parent.width
-            radius: 5
-            anchors {
-                bottom: parent.bottom
-                bottomMargin: 0
-                left: parent.left
-                leftMargin: 0
+            height: parent.height-parent.height*level
```
Comment (zfox23) [inline interface/resources/qml/hifi/simplifiedUI/inputDeviceButton/InputDeviceButton.qml]:
```suggestion
            height: parent.height - parent.height * level
```

Comment (PrestonB1123) [reply interface/resources/qml/hifi/simplifiedUI/inputDeviceButton/InputDeviceButton.qml]: Ack

File: interface/resources/qml/hifi/simplifiedUI/topBar/SimplifiedTopBar.qml (hunk 4 above)
Comment (PrestonB1123) [reply interface/resources/qml/hifi/simplifiedUI/topBar/SimplifiedTopBar.qml]: Ack

File: interface/resources/qml/hifi/simplifiedUI/topBar/SimplifiedTopBar.qml (hunk 5 above)
Comment (PrestonB1123) [reply interface/resources/qml/hifi/simplifiedUI/topBar/SimplifiedTopBar.qml]: Yes. We experimented with preserving fit and such, but it ended up doing some undesirable cropping or stretching no matter what we did.

File: interface/resources/qml/hifi/simplifiedUI/inputDeviceButton/InputDeviceButton.qml (hunk 6 above)
Comment (PrestonB1123) [reply interface/resources/qml/hifi/simplifiedUI/inputDeviceButton/InputDeviceButton.qml]: Yes. We experimented with preserving fit and such, but it ended up doing some undesirable cropping or stretching no matter what we did.

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20153/HighFidelity-Beta-PR15774-bd3fe8a.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20153/HighFidelity-Beta-Interface-PR15774-bd3fe8a.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20153/HighFidelity-Beta-PR15774-bd3fe8a.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20153/HighFidelity-Beta-Interface-PR15774-bd3fe8a.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20155/HighFidelity-Beta-PR15774-f0dd28e.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20155/HighFidelity-Beta-Interface-PR15774-f0dd28e.dmg

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20155/HighFidelity-Beta-PR15774-f0dd28e.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20155/HighFidelity-Quest-Beta-PR15774-f0dd28e.apk)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20155/HighFidelity-Beta-PR15774-f0dd28e.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20155/HighFidelity-Beta-Interface-PR15774-f0dd28e.exe
